.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: golangci-lint
```

Comment:
Let's use a better display name, maybe `lint GO code`

Reply:
Okay, I'll update it to `lint GO code`

Reply:
Thinking about it again, Is that even a valid name? maybe we want `lint-go` or `golint`

Reply:
Ok, so if there is prior work (blogs, articles) that already established a none name let revert back to it

Reply:
yes (resolving this thread)